specification/src/main/asciidoc/platform/ResourcesNamingInjection.adoc

```diff
@@ -417,8 +419,11 @@ resource injection, as well as PostConstruct and PreDestroy callbacks.
 In addition, if CDI is enabled—which it is by default—these classes also
 support CDI injection, as described in
 <<a2112, Support for Dependency
-Injection>>”, and the use of interceptors<<a3652, 5>>. The
-component classes listed with support level “Limited” only support Jakarta
+Injection>>”, and the use of interceptors.footnote:[Note that the use of
+interceptors defined by means of the Interceptors annotation is supported in the
+absence of Jakarta™ Contexts and Dependency Injection for Jakarta™ Enterprise
```

Comment:
I thought we could use CDI in the specs still?

Reply:
Agree, will update.